tests/test_integration.py

```diff
@@ -1,23 +1,23 @@
 from unittest.case import SkipTest
 
-from hyper_sh import Client
+from hyper_sh import APIClient
 
 
 def assertSetup():
     try:
-        Client.guess_config()
```

Comment:
wow. This breaks the API. We need to call it hyper_sh 2.0.0 then...

Reply:
Yeah, that makes sense.